style.css

```diff
@@ -319,37 +309,37 @@ border-radius: 8px;
   line-height: 52px;
   color: #172b4d;
   width: 20.44rem;
-  height: 6.3rem;
 }
+
 .text-2 {
   font-family: "Poppins", sans-serif;
   font-size: 16px;
   font-style: normal;
   font-weight: 400;
   line-height: 24px;
-  height: 7.13rem;
   width: 20.44rem;
   margin-top: -20px;
   color: #344563;
 }
+
 .link-2 {
   font-family: "Poppins", sans-serif;
   font-size: 15px;
   font-style: normal;
   font-weight: 500;
-  margin: 20px 0px 0px -109px;
+  margin: 0 0 0 -109px;
   text-decoration: none;
   color: #7f8cff;
 }
+
 .icons-2 {
   display: flex;
   flex-direction: row;
   align-items: center;
   justify-content: space-between;
   width: 200px;
-  height: 40px;
   list-style: none;
-  margin: -10px 0px 0px -150px;
+  margin: -10px 0 0 -150px;
 }
 
 .Button {
```

Comment:
- [ ] [REQUIRED] Good job so far! Please implement [previous reviewer suggestion](https://github.com/malembi/My-Portfolio/pull/3/files#r1107012160) 🙏🏼 Please choose the right color for your buttons using Figma for different effects(Hover, Enebled....) 🙏🏼   

![image](https://user-images.githubusercontent.com/63915024/219039309-d3bc713d-1e2e-4b37-916e-25f3cf7cacd1.png)

Reply:
made changes to the colours on the buttons as per the figma design, thanks!